downloads/models.py

```diff
@@ -248,19 +264,9 @@ def purge_fastly_download_pages(sender, instance, **kwargs):
 
     # Only purge on published instances
     if instance.is_published:
-        # Purge our common pages
-        purge_url('/downloads/')
-        purge_url('/downloads/latest/python2/')
-        purge_url('/downloads/latest/python3/')
-        purge_url('/downloads/mac-osx/')
-        purge_url('/downloads/source/')
-        purge_url('/downloads/windows/')
+        _purge_download_pages()
         if instance.get_version() is not None:
             purge_url('/ftp/python/{}/'.format(instance.get_version()))
```

Comment:
It would be nice to find a way to purge these URLs in the command as well. I think using surrogate keys may help here: https://docs.fastly.com/guides/api-caching/purging-api-cache-with-surrogate-keys (I'm not sure whetler I can create them without a Fastly account?)

Reply:
Surrogate Keys are set by simply including the header `Surrogate-Key`, no need to register them.

Reply:
Ah, I see the rub here. Since the /ftp urls are served by [another host](https://github.com/python/psf-salt/blob/196d03fe8b2b6d846032e8940a4b172d1fd82215/salt/downloads/config/nginx.downloads-backend.conf#L19-L23)... It is unfortunate that we already have a Surrogate-Key there that is static... and somewhat painful if we ever purge it (downloads box can only serve _so much_ bandwidth).